### PR TITLE
add auth register components, include store and persistegate, fix routes

### PR DESCRIPTION
**Pull request by PavloYakubovskyi**
please check.

### CODE REVIEW

**Comment by PavloYakubovskyi** [comment]
please check

File: src/components/Auth/Auth.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { NavLink, Outlet, useParams } from 'react-router-dom';
+import { Suspense } from 'react';
+import RegisterForm from 'components/Register/RegisterForm';
+
+const Auth = () => {
+  const { id } = useParams();
+
+  return (
+    <div className="welcome-wrapper">
```
Comment (NickTsalyn) [inline src/components/Auth/Auth.jsx]: make it using Styled Components, so all structure will be the same

File: src/index.js [hunk 2]
```diff
@@ -1,57 +1,59 @@
...
 
-ReactDOM.createRoot(document.getElementById("root")).render(
-	<React.StrictMode>
-		{/* <Provider store={store}> */}
-			{/* <PersistGate persistor={persistor}> */}
-				<BrowserRouter basename="NickTsalyn/task-pro-frontend">
-					<ThemeProvider theme={theme}>
-						<App >
-							<button>
-								<svg>
```
Comment (NickTsalyn) [inline src/index.js:47]: did you checked that svg icon will work? I'll send you template how it will work definitely

File: src/index.js [hunk 3]
```diff
@@ -1,57 +1,59 @@
...
+};
 
-ReactDOM.createRoot(document.getElementById("root")).render(
-	<React.StrictMode>
-		{/* <Provider store={store}> */}
-			{/* <PersistGate persistor={persistor}> */}
-				<BrowserRouter basename="NickTsalyn/task-pro-frontend">
-					<ThemeProvider theme={theme}>
-						<App >
-							<button>
```
Comment (NickTsalyn) [inline src/index.js:46]: in index.js shouldn’t be any tags, only Providers.

**Review by NickTsalyn — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 3 above)
Comment (PavloYakubovskyi) [reply src/index.js:46]: please check. Added styled-components.

**Review by NickTsalyn — APPROVED** [review]

**Pull request merged**